roles/insight/templates/dashcore-node.json

```diff
@@ -0,0 +1,25 @@
+{
+  "network": "{{ 'mainnet' if dash_network == 'mainnet' else 'testnet' }}",
```

Comment:
Shouldn't we just make this a variable? (Also, we can use a `.j2` extension if this is a standard Jinja2 template.)

Reply:
Reason is b/c neither of these options are valid if devnet, right? Or would that need more work in Insight?

Reply:
Insight doesn't know about devnet for now. And I think we don't need it yet. Devnet is kinda compatible with testnet, that's why we use testnet for all networks except mainnet. 
We will throw out from here Insight pretty soon when DAPI and wallet-library remove that dependency. 
Yes, we can make this variable, but this is temporary solution, so probably we should keep it for a while. What do you think?

Reply:
Ok, that's fine and we will throw this out anyway soon. 👍